pages/wiki/building-asteroidos.hbs

```diff
@@ -10,7 +10,7 @@ layout: documentation
 <div class="page-header">
   <h1 id="downloading">Downloading</h1>
 </div>
-<p>Start by install the prerequisites:</p>
+<p>Start by installing the prerequisites:</p>
```

Comment:
Oh... seeing it now... Next time, use git commit --amend or git rebase -i.
I'll squash your two commits into one, for clarity